src/resources/ResourceSnapshots/ResourceSnapshotsInterfaces.ts

```diff
@@ -182,6 +217,14 @@ export interface ResourceSnapshotsReportModel {
      * The result code of the snapshot event, if applicable.
      */
     resultCode: ResourceSnapshotsReportResultCode;
+    /**
+     * The ordered stages the migration operation has to go through in order to complete.
+     */
+    stagesToExecute?: SnapshotsReportStages[];
```

Comment:
Are you sure it is optional?

Reply:
@jmorissette could you answer here please

Reply:
Can confirmed we always set the value and that following https://coveord.atlassian.net/browse/SRC-5167 the list might be empty